core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -164,8 +164,11 @@ <h3>Advanced Features</h3>
                 Enabled
               </span>
             </span>
-            <span class="col-lg-8 col-md-8 col-sm-8 help-block" style="font-size: smaller;">
-              Values that change as the learner moves between cards (<a href="http://oppia.github.io/#/Parameters" target="_blank">more info</a>).
+            <span ng-if="!areParametersEnabled()" class="col-lg-8 col-md-8 col-sm-8 help-block" style="font-size: smaller;">
```

Comment:
Can you move the ng-if one level in, so that you don't need to duplicate the styles?

Also, where possible, put the ng-if only around the "disabled/enabled" sentence. Try not to duplicate the same things across branches. Ditto elsewhere.

Reply:
Done!